packages/help-extension/src/index.ts

```diff
@@ -73,6 +73,10 @@ const HELP_CLASS = 'jp-Help';
 
 const RESOURCES = [
   {
+    text: 'Jupyterlab Reference',
+    url: 'http://jupyterlab.readthedocs.io/en/stable/'
```

Comment:
Mind changing this to `https`?  We can't embed `http` if jlab itself is served via `https`.